controllers/provider-openstack/pkg/controller/controlplane/valuesprovider.go

```diff
@@ -240,6 +246,16 @@ func getConfigChartValues(
 		"requestTimeout":    cluster.CloudProfile.Spec.OpenStack.RequestTimeout,
 	}
 
+	// Only the 1.15 out of tree provider for OpenStack support the OS_DOMAIN_NAME/ID
+	// https://github.com/kubernetes/cloud-provider-openstack/pull/733
+	ok, err := gutils.CompareVersions(cluster.Shoot.Spec.Kubernetes.Version, ">=", "1.15")
```

Comment:
We cannot merge this PR yet because there is no release of the OpenStack CCM, right?

Reply:
Yes I would wait. The merge should happen tomorrow.

Reply:
Even if it's merged we have to wait for a new release. Do you know how often they are releasing?

Reply:
So they are releasing not that often. However we will push for a release once all PRs are merged (still pending) on the cloud controller side. However we could move forward with this even if the CCM isn't available yet. You are just not allowed to set the user-domain-name. :)

Reply:
What would happen if an end-user still specifies the user-domain-name field? Would the current CCM crash, or would simply happen "nothing"?

Reply:
The CCM will crashloop since the config file marshalling can't handle unknown properties. :(